ArduCopter/GCS_Copter.cpp

```diff
@@ -9,7 +9,7 @@ uint8_t GCS_Copter::sysid_this_mav() const
 
 const char* GCS_Copter::frame_string() const
 {
-    return copter.get_frame_string();
+    return copter.motors->get_frame_string();
```

Comment:
We could use AP::motors() here.  I'm not sure its worth it, but I wonder if it might allow us to move code up at some stage.

Reply:
Also, this looks like a pointless method - could just fold the call into the user.

Reply:
No longer pointless.